docs/pages/machine-workload-identity/machine-id/faq.mdx

```diff
@@ -14,6 +14,7 @@ no persistent state exists between individual workflow runs), Machine ID works
 best where a supported join method exists. These are:
 
 - GitHub Actions
+- Bitbucket
```

Comment:
Feels like we should link to the join methods docs rather than try to keep this list up to date. People aren't going to think to update the FAQ when they add a join method, but they should think to update the joining docs.